cypress/integration/specialFeatures/cookieBanner/testsForCanonicalOnly.js

```diff
@@ -72,97 +72,102 @@ const assertCookieValues = cookies => {
   });
 };
 
-const visitPage = (service, pageType) => {
-  cy.visit(config[service].pageTypes[pageType].path, {
+const visitPage = (pageType, path) => {
+  cy.visit(path, {
     failOnStatusCode: !pageType.includes('error'),
   });
 };
 
+const makeArray = arrayOrString => [].concat(arrayOrString);
+
 Object.keys(config)
   .filter(serviceFilter)
   .forEach(service => {
     Object.keys(config[service].pageTypes)
       .filter(pageType => filterPageTypes(pageType, service))
       .forEach(pageType => {
-        const { variant } = config[service];
+        const paths = makeArray(config[service].pageTypes[pageType].path);
+        paths.forEach(path => {
+          const { variant } = config[service];
 
-        describe(`Canonical Cookie Banner Test for ${service} ${pageType}`, () => {
-          it('should have a privacy & cookie banner, which disappears once "accepted" ', () => {
-            cy.clearCookies();
-            visitPage(service, pageType);
+          describe(`Canonical Cookie Banner Test for ${service} ${pageType}`, () => {
```

Comment:
Not including `path` in the description might make it a bit more difficult to diagnose failures from logs

Although including it might make the logs noiser - could maybe include it only if there is more than one path for that service | pageType?